design-proposals/enhancements.md

```diff
@@ -0,0 +1,103 @@
+# Overview
+
+Rework the existing KubeVirt design proposal process into a more
+formal - still lean - Enhancement Porposal process.
+
+## Motivation
+
+Today it's a [very small group](https://github.com/kubevirt/community/blob/main/OWNERS#L7-L14)
+which can approve design-proposals.
+While certain individuals can approve design proposals, once they are merged
+it is not clear who is owning them post-merge.
+
+In this proposal we are proposing to change the process and shape them
+around SIGs.
+At the heart this proposal is requiring
+
+1. designs to be sponsored by a SIG
+2. to make a SIG responsible for the design process (reviews of design, code, documentation)
+3. to make a SIG responsible for managing the design process (collaboration with other SIGs as needed)
+4. to make a SIG owning the feature once it has been merged. The SIG is responsible for maintaing, fixing, running, everything it.
```

Comment:
I'd suggest following k8s KEPs here so in such a case we always have an owning SIG that are responsible for the feature post merge and participating SIGs if the feature does cross SIG boundaries:

https://github.com/kubernetes/enhancements/blob/master/keps/sig-architecture/0000-kep-process/README.md#kep-metadata

Reply:
@davidvossel addressed. Now, I think we _must_ require one owning SIG, even if a fetaure is spread accross multiple. One SIG _must_ feel responsible to resolve issues or drive a feature forward, even if this requires extensive work with other SIGs.
In the end a feature has to have an owner, and it's better to have this hard discussoin through the inception of a feature - instead of waiting for a an urgency which could urge us to quickly find an owner.
Also: If we can not identify an owner for a feature, then we should ask ourselfs: Can we really take this feature if we are unable to own it?

(Just read Lee's comment now, but +1)

Reply:
Reworded in order to express ownership to be exactly 1.